Makefile

```
@@ -70,7 +70,7 @@ bump-major: bump-prep
 # Docs
 .PHONY: install-docs
 install-docs:
-	pip install -U sphinx myst-parser sphinx-book-theme
+	pip install -U sphinx myst-parser sphinx-book-theme sphinx_autodoc_typehints
```

```suggestion
	pip install -U sphinx myst-parser sphinx-book-theme sphinx-autodoc-typehints
```
I think this points to the same pypi package, but i'm a bit superstitious about package squatters so i'd rather go with the [recommended installation command](https://github.com/tox-dev/sphinx-autodoc-typehints#installation-and-setup)